cfme/configure/tasks.py

```diff
@@ -24,9 +25,12 @@
 
 buttons = Region(
     locators=dict(
-        default="//*[@id='buttons_off']/li[3]/a/img",
-        apply="//*[@id='buttons_on']/li[1]/a/img",
-        reset="//*[@id='buttons_on']/li[2]/a/img"
+        default=pick({LOWEST: "//*[@id='buttons_off']/li[3]/a/img",
+                 '5.4': "//*[@id='buttons_off']/a"}),
+        apply=pick({LOWEST: "//*[@id='buttons_on']/li[1]/a/img",
+               '5.4': "//*[@id='buttons_on']/a[1]"}),
+        reset=pick({LOWEST: "//*[@id='buttons_on']/li[2]/a/img",
+               '5.4': "//*[@id='buttons_on']/a[2]"})
```

Comment:
I'm fine with it, but can we get this merged in now, and do it in another PR? probably good to do a review of all "buttons"